src/Chainweb/Pact/Transactions/UpgradeTransactions.hs

```diff
@@ -38,3 +42,16 @@ upgradeTransactions Mainnet01 cid = case cidInt of
         cidInt = chainIdInt cid
 upgradeTransactions Development _ = Devnet.transactions
 upgradeTransactions _ _ = Other.transactions
+
+twentyChainUpgradeTransactions :: ChainwebVersion -> ChainId -> IO [ChainwebTransaction]
+twentyChainUpgradeTransactions Mainnet01 cid = case chainIdInt @Int cid of
+  0 -> MNKAD.transactions
+  c | c >= 1, c <= 9 -> return [] -- no upgrades or remeds
+  c | c >= 10, c <= 19 -> MN20.transactions
+  c -> internalError $ "Invalid mainnet chain id: " <> sshow c
```

Comment:
Single remediation tx on chain 0. 


Currently, this setup will immediately upgrade chains 10-19 with upgrades, but this is possibly not the right route to go. We may want to have 10-19 use v2 from genesis (will that even work?). 

If we want v2 to be immediately available on 10-19, it would take 5min to write an alternative workflow generating the proper genesis txs.